java/apps/recovery-portal/src/main/webapp/errors/error_400.jsp

```diff
@@ -30,9 +32,14 @@
 <%-- Branding Preferences --%>
 <jsp:directive.include file="../includes/branding-preferences.jsp"/>
 
+<%
+    String stat = IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "error.400");
```

Comment:
Let's use more descriptive variable names. check the other changed places as well.

Reply:
https://github.com/wso2/identity-apps/pull/6249/commits/015847bdf35b5e241b11121d62713211ad0950db